### DISCUSSION TITLE
interpolate() was renamed to interpolateNode() in Reanimated 2

### DISCUSSION THREAD

**Discussion by samuel-rl** [software-mansion/react-native-reanimated#2070] [Q&A]
Hello, I have just implemented reanimated 2 into a project. 

I already use react-navigation v4 with react-navigation-drawer.

```
[...]
    "react-navigation": "^4.0.10",
    "react-navigation-drawer": "^2.2.2",
    "react-navigation-stack": "^1.10.2",
[...]
```

This one still using react-native-reanimated V1.. So I got a lot of console log like this : 

```
interpolate() was renamed to interpolateNode() in Reanimated 2. Please use interpolateNode() instead 
Drawer@http://localhost:8081/index.bundle?platform=ios&dev=true&minify=false:130968:36
DrawerView@http://localhost:8081/index.bundle?platform=ios&dev=true&minify=false:129966:36
Navigator@http://localhost:8081/index.bundle?platform=ios&dev=true&minify=false:102916:38
[...]
```

Any idea to fix this ?

**Comment by NikitaDudin** [comment]
You can try to replace interpolate on interpolateNode in file node_modules/react-navigation-drawer/src/views/Drawer.tsx.
![image](https://user-images.githubusercontent.com/25579397/120015588-293d4380-bfec-11eb-89d0-6980fd476768.png)
===>
```
const {
  Clock,
  Value,
  onChange,
  clockRunning,
  startClock,
  stopClock,
  interpolateNode: interpolate,
  spring,
  abs,
  add,
  and,
  block,
  call,
  cond,
  divide,
  eq,
  event,
  greaterThan,
  lessThan,
  max,
  min,
  multiply,
  neq,
  or,
  set,
  sub,
} = Animated;
```

**Reply by samuel-rl** [reply]
Thanks, but that doesn't solve the problem, and there are a lots of logs, not only about Drawer..
